### PR TITLE
chore: Fix exit handling test suite

### PR DESCRIPTION
**Pull request by gnarea**
As in, make sure all tests pass. ESLint is reporting errors on these files.

### CODE REVIEW

File: src/utilities/exitHandling.spec.ts [hunk 1]
```diff
@@ -13,7 +13,9 @@ beforeEach(() => {
 });
 
 const mockProcessOn = mockSpy(jest.spyOn(process, 'on'));
-const mockProcessExit = mockSpy(jest.spyOn(process, 'exit'));
+const mockProcessExit = mockSpy(jest.spyOn(process, 'exit'), () => {
+  // Do nothing
```
Comment (gnarea) [inline src/utilities/exitHandling.spec.ts:17]: Found the fix here: https://stackoverflow.com/a/51448240/129437

Comment (annaavanesyan) [reply src/utilities/exitHandling.spec.ts:17]: I would be better if we omit comments, default eslint configuration is forbidding those as well [no-inline-comments]

Comment (gnarea) [reply src/utilities/exitHandling.spec.ts:17]: If I omit the comment, that'd violate another rule:

Screenshot_20230221_150928

Note that this isn't an inline comment though.

Comment (annaavanesyan) [reply src/utilities/exitHandling.spec.ts:17]: approved :)

Comment (gnarea) [reply src/utilities/exitHandling.spec.ts:17]: Thanks! Will merge

**Pull request merged**